dataflow/src/main/java/org/checkerframework/dataflow/cfg/CFGBuilder.java

```diff
@@ -401,18 +401,21 @@ public String toString() {
         protected final Node node;
 
         /**
-         * Map from exception type to labels of successors that may be reached as a result of that
-         * exception.
+         * Set of all possible pairs of the refined exception type {@link TypeMirror} and the
```

Comment:
Pairs of the refined exception **types** and the corresponding **targets' labels**?
Maybe remove `{@link Label}`? or change `label` to `{@link Label}s`.